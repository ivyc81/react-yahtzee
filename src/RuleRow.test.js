import React from 'react';
import { shallow, mount } from "enzyme";
import RuleRow from './RuleRow';
import toJson from "enzyme-to-json";

it('renders without crashing', () => {
  shallow(<RuleRow />);
});

it('renders correctly', () => {
    const wrapper = mount(<RuleRow name='test' score={ 0 }/>);
    const serialized = toJson(wrapper);
    expect(serialized).toMatchSnapshot();
});