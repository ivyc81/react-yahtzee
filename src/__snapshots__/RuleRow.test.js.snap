// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RuleRow
  name="test"
  score={0}
>
  <tr
    className="RuleRow RuleRow-active"
  >
    <td
      className="RuleRow-name"
    >
      test
    </td>
    <td
      className="RuleRow-score"
    >
      0
    </td>
  </tr>
</RuleRow>
`;
